docs/evm/rpc.md

```diff
@@ -135,7 +135,10 @@ Method | Status | Notes
 
 The Aurora Relayer source code repository is at:
```

Comment:
I can't comment on it because it's outside the range of the diff, but this file could use some updates. For example `eth_estimateGas` has been improved (it no longer returns a constant value), and Ethereum is no longer a PoW chain, so we can probably say some of the RPC methods are unsupported for that reason.

Reply:
I was going to ask about that because the whole file does need an update but I'm not entirely sure what to write about that as I'm not sure what I can say about the new infrastructure, for example, and what is wholly supported.